libs/shared/utils/src/getPaths/getPaths.ts

```diff
@@ -5,7 +5,10 @@ export const getPaths = <LinkEntry extends { isFolder: boolean; slug: string }>(
   items: LinkEntry[],
 ): TGetPaths[] =>
   items
-    .filter(({ isFolder, slug }) => slug && !isFolder && isSlugRestricted(slug))
+    .filter(
+      ({ isFolder, slug }) =>
+        slug && !isFolder && !slug.includes('/') && !isSlugRestricted(slug),
```

Comment:
This makes sense, at least for now. I don't anticipate us wanting a path hierarchy for regular site pages... easy enough to change if we *do* end up wanting it, though.